.github/workflows/main.yml

```diff
@@ -174,8 +174,14 @@ jobs:
   memcheck:
     runs-on: ubuntu-latest
     steps:
+    - name: Cache valgrind
+      uses: actions/cache@v3
+      with:
+        path: valgrind-3.20.0
+        key: ${{ runner.os }}-valgrind-3.20.0
```

Comment:
This doesn't seem to work, not sure why:
https://github.com/ruby/yarp/actions/runs/6177085624/job/16767441573?pr=1477#step:12:2